src/data/faq_de.json

```diff
@@ -2181,18 +2181,102 @@
         "permalink": "Link zu diesem Eintrag",
         "linkToGlossary": "Link zum Glossar",
         "glossaryWords": {
+            "A": [
+                {
+                    "term": "Android",
+                    "anchor": "android",
+                    "description": "Android ist neben <a href='#glossary_ios'>iOS</a> eines der beiden Betriebssysteme, welche die Corona-Warn-App aktuell unterstützt, und das am weitesten verbreitete mobile Betriebssystem der Welt. Es handelt sich um ein <a href='#glossary_open_source'>Open-Source</a> Betriebssystem für beispielsweise mobile (wie Smartphones oder Tables) aber auch andere Geräte (wie Fernseher oder Autos). Es wird gemeinsam von Google und der Open Handset Alliance (OHA) entwickelt."

```

Comment:
Tablets anstatt Tables

Reply:
Will be changed in the next commit, thanks!